controllers/agents/app/serviceclaim_controller.go

```diff
@@ -134,6 +137,15 @@ func (r *ServiceClaimReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	})
 	if err != nil {
 		if strings.Contains(err.Error(), "admission webhook \"vserviceclaim.kb.io\" denied the request") {
+			c := metav1.Condition{
+				LastTransitionTime: metav1.NewTime(time.Now()),
+				Type:               primazaiov1alpha1.ServiceBindingConditionReady,
```

Comment:
Should be `ServiceClaimConditionReady` and not `ServiceBindingConditionReady`. Applies in all places. Do you think we can created three types for different states?

Reply:
I think a common type make the condition consistent.